content/sensu-go/6.2/observability-pipeline/observe-schedule/collect-metrics-with-checks.md

```diff
@@ -16,67 +16,220 @@ menu:
+  "metadata": {
+    "created_by": "admin",
+    "name": "collect-metrics",
+    "namespace": "default"
+  },
+  "spec": {
+    "check_hooks": null,
+    "command": "metrics-disk-usage.rb",
+    "env_vars": null,
+    "handlers": [],
+    "high_flap_threshold": 0,
+    "interval": 60,
+    "low_flap_threshold": 0,
+    "output_metric_format": "",
+    "output_metric_handlers": null,
+    "proxy_entity_name": "",
+    "publish": true,
+    "round_robin": false,
+    "runtime_assets": [
+      "disk-checks-plugins",
+      "sensu-ruby-runtime"
+    ],
+    "secrets": null,
+    "stdin": false,
+    "subdue": null,
+    "subscriptions": [
+      "linux"
+    ],
+    "timeout": 0,
+    "ttl": 0
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+## Extract metrics from check output
+
+To extract metrics from check output, configure the check `output_metric_format` to one of the [supported output metric formats][3] (in this example, InfluxDB Line Protocol):
+
+{{< code shell >}}
+sensuctl check set-output-metric-format collect-metrics influxdb_line
+{{< /code >}}
+
+You should receive a confirmation response: `Updated`.
+
+To view the updated check definition and confirm that it specifies `influxdb_line` as the output metric format, run:
+
+{{< language-toggle >}}
+
+{{< code shell "YML" >}}
+sensuctl check info collect-metrics --format yaml
+{{< /code >}}
+
+{{< code shell "JSON" >}}
+sensuctl check info collect-metrics --format wrapped-json
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+## Confirm that your check is collecting metrics
+
+Use sensuctl or log into the Sensu web UI to confirm that your `collect-metrics` check is working.
+
+To use sensuctl:
+
+{{< code shell >}}
+sensuctl event list
+{{< /code >}}
+
+You should see an event for the `collect-metrics` check for every entity with the `linux` subscription:
+
+{{< code shell >}}
+      Entity           Check                                           Output                                             Status   Silenced             Timestamp                             UUID                  
+ ─────────────── ───────────────── ───────────────────────────────────────────────────────────────────────────────────── ──────── ────────── ─────────────────────────────── ────────────────────────────────────── 
+  i-424242-test   collect-metrics   metrics-disk-usage.rb last sent from i-424242-test at 2021-02-08 14:55:10 -0500 EST        0   false      2021-02-08 14:55:10 -0500 EST   63c91615-6438-4f24-992b-04f616d6f917  
+{{< /code >}}
 
-See [Troubleshoot Sensu][10] for an example debug handler that writes events to a file for inspection.
+To confirm in the web UI, open `http://localhost:3000/`.
+Navigate to the **Events** page and search for passing checks: `event.check.state == passing`.
+The `collect-metrics` check should be listed on that page.
 
-The example check would yield an event similar to this:
+The `collect-metrics` check yields an event similar to this example:
```

Comment:
I haven't replaced the original event example yet because my metrics check is failing. I think I'm running into the error described at https://docs.sensu.io/sensu-go/latest/operations/maintain-sensu/troubleshoot/#conflating-operating-systems-with-families.